charts/corda/templates/crypto-worker.yaml

```
@@ -35,8 +36,7 @@ spec:
       - name: {{ include "corda.workerName" . | quote }}
         image: {{ include "corda.workerImage" . }}
         imagePullPolicy:  {{ .Values.imagePullPolicy }}
-        securityContext:
-          allowPrivilegeEscalation: false
+        {{- include "corda.workerConainterSecurityContext" . | nindent 8 }}

```

And in all the other workers...
```suggestion
        {{- include "corda.workerContainerSecurityContext" . | nindent 8 }}

```